### DISCUSSION TITLE
I have some existing code I want to submit to be reviewed by GPT-engineer, is there a way to do this without copying it all into a really big prompt

### DISCUSSION THREAD

**Discussion by geewhizbang** [AntonOsika/gpt-engineer#494] [Q&A]
I have some VueJS / Typescript code that works, but I would like GPT-Engineer to review it and make suggestions. Is there a way to do this other than copying all the code into a really big prompt?

**Comment by lmagee3** [comment]
Yea, I was wondering that too, and you totally can. You can essentially have GPT-Engineer look at files from another folder and use them in the creation of a new project. You can do this by adding the path to the folder to the files parameter in the prompt file. For example, if you have a folder called my-custom-files that contains some code files, you would add the following line to the prompt file: files: ["my-custom-files"]
This will tell GPT-Engineer to look at the files in the my-custom-files folder and use them in the creation of the new project. Make sure that the file folder is in the same directory as your project.

Here is an example of how the  prompt file that uses files from another folder should look:

 name: "My New Project"
description: "This is a new project that uses files from another folder."
files: ["my-custom-files"]

Just change the 'name' to whatever the name of your project is, and the files: [to whatever the file name is]
then just 'gpt-engineer project/whatever your project name is' in your terminal or CMD and mind-blown
